ci/rebuild_updated_recipes.py

```diff
@@ -65,34 +65,65 @@ def build(target_python, requirements):
     with current_directory('testapps/'):
         # iterates to stream the output
         for line in sh.python(
-                testapp, 'apk', '--sdk-dir', android_sdk_home,
-                '--ndk-dir', android_ndk_home, '--bootstrap', 'sdl2', '--requirements',
-                requirements, _err_to_out=True, _iter=True):
+                testapp, 'apk',
+                '--sdk-dir', android_sdk_home,
+                '--ndk-dir', android_ndk_home,
+                '--bootstrap', target_bootstrap,
+                '--requirements', requirements,
+                _err_to_out=True, _iter=True):
             print(line)
 
 
-def main():
-    target_python = TargetPython.python3
-    recipes = modified_recipes()
-    logger.info('recipes modified: {}'.format(recipes))
-    recipes -= CORE_RECIPES
-    logger.info('recipes to build: {}'.format(recipes))
+def get_bootstrap(recipes_and_target):
+    """
+    Finds the right bootstrap given a set of requirements with a defined
+    target python recipe inside this set.
+    """
     context = Context()
-    # forces the default target
-    recipes_and_target = recipes | set([target_python.name])
+    bootstrap = None
     try:
         build_order, python_modules, bs = get_recipe_order_and_bootstrap(
             context, recipes_and_target, None)
+        bootstrap = bs.name
     except BuildInterruptingException:
-        # fallback to python2 if default target is not compatible
-        logger.info('incompatible with {}'.format(target_python.name))
-        target_python = TargetPython.python2
-        logger.info('falling back to {}'.format(target_python.name))
+        pass
+    return bootstrap
+
+
+def main():
+    target_python_priorities = [
+        TargetPython.python3,
+        TargetPython.python2,
+        TargetPython.python2legacy
```

Comment:
`python2legacy` is unlikely to be used unless we modify some `pygame` dependant recipe, which will be removed soon, so...I'm not sure if we have to introduce this :thinking:

Reply:
Yes by default I would be lazy and drop

Reply:
Yes, but we already have the work done in here, so maybe we should maintain this, at least until @inclement can finish his `pygame removal work`, then maybe we could start to remove all the `python2legacy` stuff. It's easy to remove this, I'm more concerned with other `python2legacy` code...but we will add some more code to remove...mmm...@inclement, what is your opinion on this?

Reply:
Ok, after thinking about it, I took the lazy road :grin:, removed `python2legacy`